### PR TITLE
Experiment scripts for frequency sweep

### PR DESCRIPTION
**Pull request by dianarg**
- Frequency sweep helpers and run script for DGEMM
- Runtime/energy vs frequency plot based on frequency sweep

### CODE REVIEW

File: integration/apps/geopmbench.py [hunk 1]
```diff
@@ -0,0 +1,76 @@
+import os
```
Comment (bgeltz) [inline integration/apps/geopmbench.py]: Missing license header.

File: integration/apps/geopmbench.py [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+import textwrap
+
+from . import apps
```
Comment (bgeltz) [inline integration/apps/geopmbench.py]: Why not just ```import apps```?

**Review by bgeltz — COMMENTED** [review]
I think I'd benefit from a review session on all of this.  I didn't realize there was so much new code.

File: integration/apps/geopmbench.py (hunk 2 above)
Comment (dianarg) [reply integration/apps/geopmbench.py]: That pattern didn't work unless CWD was in the PYTHONPATH.  We probably need to look through all the files and make sure we import in a consistent way.  There are also some differences between python2 and 3.

**Comment by dianarg** [comment]
This PR is one patch on top of #1233

**Review by cmcantalupo — DISMISSED** [review]
LGTM.

**Review by cmcantalupo — DISMISSED** [review]

**Review by cmcantalupo — APPROVED** [review]

**Pull request merged**